recipes/trantor/all/conanfile.py

```diff
@@ -77,17 +84,22 @@ def validate(self):
         if is_msvc(self) and self.options.shared and "MDd" in msvc_runtime_flag(self):
             raise ConanInvalidConfiguration(f"{self.ref} does not support the MDd runtime on Visual Studio.")
 
+        if "with_spdlog" in self.options and \
+            not self.dependencies["spdlog"].options.header_only:
+            raise ConanInvalidConfiguration(f"{self.ref} requires header_only spdlog.")
+
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
     def generate(self):
         tc = CMakeToolchain(self)
-        if Version(self.version) < "1.5.6":
-            tc.variables["BUILD_TRANTOR_SHARED"] = self.options.shared
-        else:
-            # Trantor's CMakeLists.txt has BUILD_SHARED_LIBS option.
-            tc.variables["BUILD_SHARED_LIBS"] = self.options.shared
+        # Trantor's CMakeLists.txt has BUILD_SHARED_LIBS option.
+        tc.variables["BUILD_SHARED_LIBS"] = self.options.shared
```

Comment:
please remove this and bump `required_conan_version` to 1.54.0

Reply:
@SpaceIm 
Thank you for your comment!
I update this PR.
Please review it again.